pkg/domain/transfers/repository.go

```diff
@@ -0,0 +1,18 @@
+package transfers
+
+import (
+	"context"
+
+	"github.com/daniel1sender/Desafio-API/pkg/domain/entities"
+)
+
+type Repository interface {
+	Insert(ctx context.Context, transfer entities.Transfer) error
+	ListByID(ctx context.Context, accountID string) ([]entities.Transfer, error)
```

Comment:
```suggestion
	ListByAccountID(ctx context.Context, accountID string) ([]entities.Transfer, error)
```
`ListByID` dá a ideia que ce tá listando pelo ID da transferência, o que seria incoerente pois ao listar por ID o retorno esperado seria de apenas uma ocorrência. Essa mesma lógica se aplica a outros métodos correlatos